
Add new function trace_sched_stat_register() to enable schedstats for trace events.

diff -rup linux-4.4.13/kernel/sched/bfs.c linux-4.4.13.new/kernel/sched/bfs.c
--- linux-4.4.13/kernel/sched/bfs.c	2016-06-08 11:33:21.132487524 +0200
+++ linux-4.4.13.new/kernel/sched/bfs.c	2016-06-08 11:47:50.668883496 +0200
@@ -200,6 +200,13 @@ int sysctl_schedstats(struct ctl_table *
 #endif
 #endif
 
+void trace_sched_stat_register(void)
+{
+#ifdef CONFIG_SCHEDSTATS
+	force_schedstat_enabled();
+#endif
+}
+
 void print_scheduler_version(void)
 {
 	printk(KERN_INFO "BFS CPU scheduler v0.467 by Con Kolivas.\n");
